src/uwtools/tests/config/formats/test_sh.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=duplicate-code
+"""
+Tests for uwtools.config.formats.sh module.
+"""
+
+import filecmp
+from typing import Any, Dict
+
+from uwtools.config.formats.sh import SHConfig
+from uwtools.tests.support import fixture_path
+
+# Tests
+
+
+def test_parse_include():
+    """
+    Test that non-YAML handles include tags properly for bash with no sections.
```

Comment:
I mentioned it in the library code, too, but I think we should use `sh` instead of `bash` as the latter is too specific. We're really talking about the entire Bourne-shell family, which includes `sh`, `bash`, `ksh`, `dash`, maybe `zsh`. `sh` is a good generic identifier for all those, since it refers to the original Bourne shell.

Reply:
Mmm, very good point